src/Juvix/Compiler/Backend/Geb/Pretty/Values/Base.hs

```diff
@@ -0,0 +1,97 @@
+module Juvix.Compiler.Backend.Geb.Pretty.Values.Base where
```

Comment:
Wouldn't it be better if we just added the instances from this file to Pretty/Base.hs? This would allow us to delete Pretty/Values.hs and avoid much code duplication.

Reply:
Done.